Composer/packages/server/src/models/connector/csharpBotConnector.ts

```diff
@@ -222,7 +183,9 @@ export class CSharpBotConnector implements IBotConnector {
     });
   };
 }
-
+process.on('uncaughtException', err => {
+  log(err);
+});
```

Comment:
Probably not a place for this. It doesn't make sense in this module.

Reply:
removed.